src/ui/SmartSnippet/SmartSnippet.ts

```diff
@@ -159,8 +183,11 @@ export class SmartSnippet extends Component {
   }
 
   private buildHeightLimiter() {
-    return (this.heightLimiter = new HeightLimiter(this.shadowContainer, this.snippetContainer, 400, isExpanded =>
-      isExpanded ? this.sendExpandSmartSnippetAnalytics() : this.sendCollapseSmartSnippetAnalytics()
+    return (this.heightLimiter = new HeightLimiter(
```

Comment:
There could be a very simple test that verify that the option is respected by the component, as simple as it might seem.

Reply:
Sure, I'll add a test.